ReplayBrowser/Helpers/GitHubApiHelper.cs

```diff
@@ -0,0 +1,113 @@
+                    response.EnsureSuccessStatusCode();
+
+                    var responseBody = await response.Content.ReadAsStringAsync();
+                    
+                    // Deserialize JSON into GitHubAccount list
+                    var responseJson = JsonSerializer.Deserialize<List<JsonElement>>(responseBody);
+
+                    foreach (var obj in responseJson)
+                    {
+
+                        // Initialize variables with default values
+                        // Doing this to 100% make sure that respinse errors / null responses don't crash anything
+                        string accountName = null;
+                        string accountImageURL = null;
+                        string accountLink = null;
+
+                        // Safely retrieve properties
+                        if (obj.TryGetProperty("login", out var loginProperty))
+                        {
+                            accountName = loginProperty.GetString();
+                        }
+
+                        if (obj.TryGetProperty("avatar_url", out var avatarUrlProperty))
+                        {
+                            accountImageURL = avatarUrlProperty.GetString();
+                        }
+
+                        if (obj.TryGetProperty("html_url", out var htmlUrlProperty))
+                        {
+                            accountLink = htmlUrlProperty.GetString();
+                        }
+
+                        // Check for nulls before adding to the list
+                        if (accountName != null && accountImageURL != null && accountLink != null)
+                        {
+                            var account = new GitHubAccount
+                            {
+                                AccountName = accountName,
+                                AccountImageURL = accountImageURL,
+                                AccountLink = accountLink
+                            };
+
+                            Contributors.Add(account);
+                        }
+                        else
+                        {
+                            Log.Warning("A required property was missing in the JSON response while attempting to fetch project contributors.");
+                        }
+                    }
+
+                    Log.Information("Successfully fetched project contributor list.");
+                    return Contributors;
+                }
+            }
+        }
+        catch (Exception e)
+        {
+            Log.Error($"Exception when querying GitHub: {e.Message} - {e.StackTrace.ToString()}");
+            
+            // return empty list
+            // TODO - Proper handling
+            return new List<GitHubAccount>();
+        }
+    }
+}
+
+public struct GitHubAccount
+{
+    public string AccountName { get; set; }
+    public string AccountImageURL { get; set; }
+    public string AccountLink { get; set; }
```

Comment:
Slight nitpick, you should make these `init` instead of `set` if you aren't intending to change the data after the object is created.